prototype_source/fx_graph_mode_ptq_static.rst

```diff
@@ -75,7 +75,7 @@ Download the `torchvision resnet18 model <https://github.com/pytorch/vision/blob
     import sys  
     import torch.quantization   
 
-    # Setup warnings    
+    # Set up warnings
     import warnings 
     warnings.filterwarnings(    
         action='ignore',    
```

Comment:
please also update all `torch.quantization` in this file to `torch.ao.quantization`, another example is in L87